docs/docs/setting-up-gatsby-without-gatsby-new.md

```diff
@@ -0,0 +1,66 @@
+## Setting Up a Gatsby Site Without Gatsby New
```

Comment:
```suggestion
## Setting Up a Gatsby Site Without `gatsby new`
```
@LekoArts can we use syntax highlighting in frontmatter?

Reply:
Yep, should work